src/Tasks.UnitTests/Move_Tests.cs

```diff
@@ -461,8 +461,7 @@ public void OutputsOnlyIncludeSuccessfulMoves()
         /// Moving a locked file will fail
         /// </summary>
         [Fact]
-        [Trait("Category", "netcore-osx-failing")]
-        [Trait("Category", "netcore-linux-failing")]
+        [PlatformSpecific(TestPlatforms.Windows)] // "File names under Unix are case-sensitive and this test is not useful"
```

Comment:
```suggestion
        [PlatformSpecific(TestPlatforms.Windows)] // "File locking Unix differs significantly from Windows"
```

Reply:
errr.. I thought I had removed that, but turns out it was in the mono commit.